invenio_app_ils/documents/mappings/os-v1/documents/document-v1.0.0.json

```diff
@@ -2,24 +2,48 @@
   "settings": {
     "analysis": {
       "normalizer": {
-        "lowercase_normalizer": {
+        "custom_normalizer": {
           "type": "custom",
-          "char_filter": [],
+          "char_filter": [
+            "strip_special_chars"
```

Comment:
just a question to clarify: will this strip the characters when sending the search query or when indexing?

Reply:
Since I mention this in the custom_analyzer which I mention for both index analysing and search analysing, so both!